evm/src/helpersV2.ts

```diff
@@ -271,13 +271,23 @@ export async function fulfillOracleRequest(
     gasLimit: 1000000, // FIXME: incorrect gas estimation
   },
 ): ReturnType<typeof oracleContract.fulfillOracleRequest> {
+  const d = debug.extend('fulfillOracleRequest')
```

Comment:
is the use of 'd' in typescript the norm?  Maybe a longer name would be better, even if its use is close to the declaration.

Reply:
I dont think there is a norm here, I tend to make debugging stuff with short variable names since it adds visual noise and detracts from actual logic. Let me know if you think it should be changed anyway if you think its hard to read